.github/workflows/deploy-containers.yml

```diff
@@ -27,133 +45,55 @@ jobs:
-
-      - name: Deploy image for retrieval service
-        if: steps.wait-for-build.outputs.conclusion == 'success'
-        timeout-minutes: 10
-        uses: aws-actions/amazon-ecs-deploy-task-definition@7218b9c
-        with:
-          task-definition: ${{ steps.taskdef-retrieval.outputs.task-definition }}
-          service: KeyRetrieval
-          cluster: ${{ steps.cluster.outputs.name }}
-          wait-for-service-stability: true
-          codedeploy-appspec: ${{ github.workspace }}/covidshield-retrieval-appspec.json
-
-      - name: Logout of Amazon ECR
-        if: always()
-        run: docker logout ${{ steps.login-ecr.outputs.registry }}
-
-  deploy-submission-service:
-    if: github.ref == 'refs/heads/master' && github.event_name == 'push'
-    runs-on: ubuntu-latest
-    steps:
-      - name: Wait for container to be built and pushed
-        uses: fountainhead/action-wait-for-check@v1.0.0
-        id: wait-for-build
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          checkName: build-n-push
-          ref: ${{ github.sha }}
-
-      - name: Checkout
-        if: steps.wait-for-build.outputs.conclusion == 'success'
-        uses: actions/checkout@v2
-
-      - name: Configure AWS credentials
-        if: steps.wait-for-build.outputs.conclusion == 'success'
-        uses: aws-actions/configure-aws-credentials@v1
-        with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-region: ca-central-1
-
-      - name: Login to Amazon ECR
-        if: steps.wait-for-build.outputs.conclusion == 'success'
-        id: login-ecr
-        uses: aws-actions/amazon-ecr-login@v1
-
       - name: Get Cluster Name
         if: steps.wait-for-build.outputs.conclusion == 'success'
         id: cluster
         run: |
           echo "##[set-output name=name;]$(aws ecs list-clusters | jq -r '.clusterArns[0]' | cut -f 2 -d "/" )"
 
-      - name: Download submission task definition
+      - name: Download task definition
         if: steps.wait-for-build.outputs.conclusion == 'success'
-        id: download-taskdef-submission
+        id: download-taskdef
         run: |
-          aws ecs describe-task-definition --task-definition KeySubmission --query taskDefinition > task-definition-submission.json
-          echo "##[set-output name=container_name;]$(jq -r '.containerDefinitions[0].name' task-definition-submission.json)"
+          aws ecs describe-task-definition --task-definition ${{ matrix.SERVICE_NAME }} --query taskDefinition > ${{ matric.TASK_DEFINITION_FILE }}
+          echo "##[set-output name=container_name;]$(jq -r '.containerDefinitions[0].name' ${{ matric.TASK_DEFINITION_FILE }}"
```

Comment:
Same on this line... thinking `matric` should be `matrix`

Reply:
Fixed